### ISSUE TITLE
list with no-lines inside menu still add the border line to items

### ISSUE THREAD

**Issue by Abderrahmane-H** [driftyco/ionic#11414]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
the no-lines attribute is not working when added to a list inside a menu (ios and android).
![screenshot from 2017-04-27 17-25-34](https://cloud.githubusercontent.com/assets/22124417/25493617/b57f75a0-2b6e-11e7-935b-a818c2018b6a.png)

**Expected behavior:**
the item lines must be hidden when the no-lines attribute is added to the list

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.0.1
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.4
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Linux 4.10
Node Version: v6.10.2
Xcode version: Not installed
```

**Comment by Abderrahmane-H** [comment]
## Update :
this happens only with "ionic serve"

**Comment by jgw96** [comment]
Thanks for using Ionic, could you share a repo we could use to reproduce this issue?

**Comment by Abderrahmane-H** [comment]
I am closing this as it is fixed in the latest version 3.1.0. I was using the version 3.0
